doc/examples/qubitekk/ex_qubitekk_gui.py

```diff
@@ -87,7 +87,7 @@ def reset(*args):
     # open connection to coincidence counter. If you are using Windows, this will be a com port. On linux, it will show
     # up in /dev/ttyusb
     if _platform == "linux" or _platform == "linux2":
-        cc = ik.qubitekk.CC1.open_serial('/dev/ttyUSB0', 19200, timeout=10)
+        cc = ik.qubitekk.CC1.open_serial('/dev/ftdi_xxxxxx', 19200, timeout=10)
```

Comment:
I think I'd rather switch it to finding the port based on the vid/pid/serial number like I did in the other example.